extensions/PhabBugz/Extension.pm

```diff
@@ -12,8 +12,26 @@ use strict;
 use warnings;
 use parent qw(Bugzilla::Extension);
 
+use Bugzilla::Constants;
+use Bugzilla::Extension::PhabBugz::Feed;
+use Bugzilla::Extension::PhabBugz::Logger;
+
 our $VERSION = '0.01';
 
+BEGIN {
+    *Bugzilla::phabbugz_ext = \&_get_instance;
+}
+
+sub _get_instance {
+    my $cache = Bugzilla->request_cache;
```

Comment:
You're storing this in the request cache, but the daemon isn't ever clearing the request cache. What is the lifecycle of this object intended to be in the daemon? Always there?